modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/Synteny_conf.pm

```diff
@@ -51,11 +54,11 @@ sub default_options {
     my ($self) = @_;
     return {
         %{$self->SUPER::default_options},   # inherit the generic ones
-
-        'master_db' => 'compara_master',
-        'work_dir'  => $self->o('pipeline_dir'),
+        'division'     => 'plants',
+        
+        'alignment_db'    => 'compara_curr',
+        'curr_release_db' => 'compara_curr',
```

Comment:
These seem like the kind of thing you would have moved to the base class in other cases (since the setting is the same for verts and plants). Any reason this is different?